apps/prairielearn/src/sync/syncFromDisk.ts

```diff
@@ -191,6 +191,14 @@ export async function syncDiskToSql(
   courseDir: string,
   logger: ServerJobLogger,
 ): Promise<SyncResults> {
+  if (!config.verboseSync) {
+    logger = {
+      info: logger.info.bind(logger),
+      error: logger.error.bind(logger),
+      warn: logger.warn.bind(logger),
+      verbose: () => {},
+    };
+  }
```

Comment:
An alternative is to have a conditional on every `logger.verbose()` call, but I think that may be less future-proof. Another option I tried was to add a function in `ServerJobLogger` to determine the verbosity level, but the issue there is that other code that depends on that class relies on compatibility with other loggers like winston, so it would be a bit tricky to set that up.